pkg/server/requestinfo/kcp_request_info_resolver.go

```diff
@@ -33,16 +34,28 @@ func NewKCPRequestInfoResolver() *KCPRequestInfoResolver {
 	}
 }
 
-var clustersRE = regexp.MustCompile(`/clusters/[^/]+/(.*)$`)
+var clustersRE = regexp.MustCompile(`/clusters/[^/]+(/.*)$`)
```

Comment:
am curious: why don't we start with `^` aka match the beginning of the string? Now we would also match `/foo/cluster/...`. Don't think that's intentional.

Reply:
>  am curious: why don't we start with ^ aka match the beginning of the string? 

This is to handle all `/services/...` URLs. The request resolver was actually added to handle the `/services/...` URLs - https://github.com/kcp-dev/kcp/commit/963ffdeb5818f1c125b6e1dd8e28f05127062228

Some more additional context is in https://github.com/kcp-dev/kcp/issues/2479#issuecomment-1404177658

Reply:
the service URLs rechnically have nothing to do with the main kcp server. Would rather expect them to strip the prefix first before apply this resolver.

Reply:
Im bit lost. Will this solve :ncdc: comment about 60s timeouts? As if I want to build service with streaming to VW backed services, this might be an issue.?

Reply:
So expectations is that WE ALWAYS trim first prefix for `/services`, and first `/clusters`?